themes/default-bootstrap/order-detail.tpl

```diff
@@ -406,6 +406,7 @@
 			</thead>
 			<tbody>
 			{foreach from=$messages item=message name="messageList"}
+				{if !(!$message.id_customer && !$message.id_employee)}
```

Comment:
should it really be **if !(** ?
Can you just do if $message.id_customer && $message.id_employee

Or am I missing something?